### PR TITLE
the power of the M E D I C A L   W R E N C H now makes you glow

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

medical wrench now makes you glow

## Why It's Good For The Game

good change

## Changelog
:cl:
add: medical wrench now makes you G L O W
/:cl:

### CODE REVIEW

**Comment by ecronogist** [comment]
why

**Comment by itseasytosee** [comment]
why

**Comment by Qustinnus** [comment]
Fuck phone misclick

**Comment by ecronogist** [comment]
oooooo qustinnus out here closing pull requests hehe......

**Comment by Fikou** [comment]
> Fuck phone misclick

bye bye issue jannie

File: code/game/objects/items/tools/wrench.dm [hunk 1]
```diff
@@ -79,11 +79,10 @@
 
 /obj/item/wrench/medical/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] is praying to the medical wrench to take [user.p_their()] soul. It looks like [user.p_theyre()] trying to commit suicide!</span>")
-	// TODO Make them glow with the power of the M E D I C A L W R E N C H
```
Comment (Borisvanmemes) [inline code/game/objects/items/tools/wrench.dm]: yes, good.

Comment (Fikou) [reply code/game/objects/items/tools/wrench.dm]: good

**Comment by SealCure** [comment]
Fuck chems and rebalances, these are the quality PRs we need

**Comment by Potato-Masher** [comment]
I can finally sacrafice myself to my chosen diety, fuck yeabthis is why we have the antag freeze.

**Comment by 81Denton** [comment]
Dare I say.... based??

**Review by 81Denton — CHANGES_REQUESTED** [review]
In wrench.dm, replace `materials` with `custom_materials` on lines 14+52 and this is good to merge

**Pull request merged**